apps/docs/src/app/platform/component-docs/platform-forms/platform-form-generator/platform-form-generator-docs.module.ts

```diff
@@ -0,0 +1,53 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { ApiComponent } from '../../../../documentation/core-helpers/api/api.component';
+import { API_FILES } from '../../../api-files';
+import { SharedDocumentationPageModule } from '../../../../documentation/shared-documentation-page.module';
+import { PlatformCheckboxModule, FdpFormGroupModule, PlatformButtonModule, PlatformSliderModule, PlatformFormGeneratorModule } from '@fundamental-ngx/platform';
```

Comment:
please use the appropriate imports path like this
```
import { PlatformCheckboxModule } from '@fundamental-ngx/platform/checkbox';
```

Reply:
I guess `platform` does not support it yet

Reply:
Platform does not support deep imports yet.